schemas/org.gnome.shell.extensions.x11gestures.gschema.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schemalist>
+  <schema id="org.gnome.shell.extensions.x11gestures" path="/org/gnome/shell/extensions/x11gestures/">
```

Comment:
I believe that it'd be better to add the compiled schema to the `.gitignore` and build it in `.github/workflows/release.yaml`. It'd be nice to document how to compile it as well in `HACKING.md`.

Reply:
I'm definitely familiar with GitHub's workflows so I will try to apply this change as soon as I can.
Obviously feel free to do it yourself in case you have time 🙂

Reply:
I updated the workflow `release.yml`